### PR TITLE
Update v1_26.md

### PR DESCRIPTION
**Pull request by JamesBauman**
added v1.26 changes

Signed-off-by: jamesbauman <james.bauman@broadcom.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

**Comment by JamesBauman** [comment]
@IgorCATech -  Hi Igor -- i requested a commit into master in error.  How does one cancel a commit?

**Review by JamesBauman — COMMENTED** [review]
@IgorCATech Can you please review and approve this PR?   Or do i need to ask Ashley to approve it?

**Comment by IgorCATech** [comment]
Going to look into it, Jim!

File: docs/getting-started/release-notes/v1_26.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+* Fixed an issue where plug-in installation and uninstallation did not work with NPM version 8. [#683](https://github.com/zowe/imperative/issues/683)
+
+The following bugs were fixed in the **Imperative CLI Framework**:
+
+* Fixed an issue where plugin installation and uninstallation did not work with NPM version 8. [#683](https://github.com/zowe/imperative/issues/683)
+
+The following bugs were fixed in the **CA View Plug-in for Zowe CLI**
+
+* Addressed several vulnerabilities in dependencies.
```
Comment (IgorCATech) [inline docs/getting-started/release-notes/v1_26.md:64]: Maybe consider adding a PR or an issue number and link

File: docs/getting-started/release-notes/v1_26.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+* Fixed incorrect values that are displayed for the report `Pages` and `TotalPages` fields.
```
Comment (IgorCATech) [inline docs/getting-started/release-notes/v1_26.md:66]: Same here

**Review by IgorCATech — COMMENTED** [review]
Just some minor comments. Otherwise, looks good to me!

**Comment by JamesBauman** [comment]
@IgorCATech  - Hi Igor, I inquired about the PR/issue numbers for CA View with the team.  The answer that i got is:

"The CA View Plug-in is not in a public repository so the issue numbers do not provide any value"

So i will not be adding the info that you suggested...

Thx,
jb

**Comment by IgorCATech** [comment]
> @IgorCATech - Hi Igor, I inquired about the PR/issue numbers for CA View with the team. The answer that i got is:
> 
> "The CA View Plug-in is not in a public repository so the issue numbers do not provide any value"
> 
> So i will not be adding the info that you suggested...
> 
> Thx, jb

Makes sense :) So, in this case, you can't add the links - they won't be accessible.

**Review by IgorCATech — APPROVED** [review]

**Pull request merged**